### ISSUE TITLE
Missing initial `aria-label` on DarkMode `<button>`

### ISSUE THREAD

**Issue by markteekman** [markteekman/accessible-astro-components#18]
There needs to be an `aria-label` added to the initial `<button>` element so it's state is announced to screen readers.
